frontend/package.json

```diff
@@ -54,18 +54,18 @@
   "dependencies": {
     "@openshift/dynamic-plugin-sdk": "^4.0.0",
     "@openshift/dynamic-plugin-sdk-utils": "^4.0.1",
-    "@patternfly/patternfly": "^5.2.1",
+    "@patternfly/patternfly": "^5.3.1",
     "@patternfly/quickstarts": "^5.1.0",
     "@patternfly/react-catalog-view-extension": "^5.0.0",
-    "@patternfly/react-charts": "^7.1.1",
-    "@patternfly/react-code-editor": "^5.2.1",
-    "@patternfly/react-core": "^5.2.1",
+    "@patternfly/react-charts": "^7.3.1",
+    "@patternfly/react-code-editor": "^5.3.3",
+    "@patternfly/react-core": "^5.3.3",
     "@patternfly/react-drag-drop": "^5.3.3",
-    "@patternfly/react-icons": "^5.2.1",
+    "@patternfly/react-icons": "^5.3.2",
     "@patternfly/react-log-viewer": "^5.2.0",
-    "@patternfly/react-styles": "^5.2.1",
-    "@patternfly/react-table": "^5.2.1",
-    "@patternfly/react-tokens": "^5.2.1",
+    "@patternfly/react-styles": "^5.3.1",
+    "@patternfly/react-table": "^5.3.3",
+    "@patternfly/react-tokens": "^5.3.1",
     "@patternfly/react-topology": "^5.4.0-prerelease.6",
     "@patternfly/react-virtualized-extension": "^5.0.0",
```

Comment:
Upgrade this to 5.1.0 https://npmjs.com/package/@patternfly/react-virtualized-extension

Reply:
@dpanshug I referred this link  https://www.patternfly.org/get-started/release-highlights , I see react-virtualized-extension promoted package version is still `5.0.0`
cc @christianvogt

Reply:
Maybe it was not mentioned in release highlights but the latest according to [github release](https://github.com/patternfly/react-virtualized-extension/releases/tag/v5.1.0) is 5.1.0